script/Ciel/Day 8/08_01_CIEL08_1D.txt

```diff
@@ -430,57 +483,63 @@ In front of me, obviously, lies the corpse.
 [sha:d2f020f1e970689863d9dc30a81ffa7ef6e0aea1]{
 -- Page 127, Offset 18592.
 -- 人体は動かなくなると目障りなだけの荷物になる事を知った。
-I know that a human body that no longer moves is nothing more than an eye-sore.
+I know that once a human body ceases to move, it becomes nothing more than an eyesore. Dead weight.
 }
 [sha:f255556b41c2ee31b49d7614be49651f5d37806d]{
 -- Page 127, Offset 18593.
 -- とにかく場所をとり、始末に困り、処理に手間取る。
-It takes up space and disposing of it is troublesome and takes a while.
+It takes up space, it's difficult to dispose of, and tedious to clean up after.
 }
 [sha:db5f2f6b0903f3bfc220a75e95e010b090b768dc]{
 -- Page 127, Offset 18594.
 -- なるほど。室内でコレが好まれないのは当然だ。
-I see. It's clear why people don't like having that indoors.
+I see. No wonder people don't like them lying around the house.
 }
 [sha:44eaa3f3346b99be6882fcc51fb4129d8fa9122f]{
 -- Page 128, Offset 18595.
 -- “相手が蘇生したから罪はない？　論点を逸らすなよ。相手がなんであれ、おまえが異常である事に変わりはない”
-'It's not a sin if she revives? You're just evading the problem. No matter who you killed, that doesn't change your abnormality.'
+'It's not a sin if she revives?
+ Don't try and dodge the point.
+ The details of who you killed don't matter, you're still a freak.'
```

Comment:
> 'Did you really think you were devoid of sin just because she came back to life?
That's was never the point.
The details of who you killed don't matter.
It doesn't take away from how fucked up you are.'